src/main/resources/messages_cy.properties

```diff
@@ -127,6 +127,9 @@ public.login.error.locked.title=Mae problem gyda’ch manylion mewngofnodi
 public.login.error.locked.instruction=Mae eich cyfrif wedi’i gloi oherwydd gormod o ymdrechion aflwyddiannus i ddefnyddio'ch cyfrif.<br>Gallwch  <a href="{0}">ailosod eich cyfrinair</a> i ddatgloi eich cyfrif.
 public.login.error.suspended.title=Mae problem gyda’ch manylion mewngofnodi
 public.login.error.suspended.instruction=Mae eich cyfrif wedi cael ei flocio.  Cysylltwch â ni i gael cymorth i fewngofnodi. 
```

Comment:
I dont mind this if we have a ticket in the sprint to make sure we fix this before we proceed

Reply:
@nikola-naydenov-hmcts 
Created a ticket in the backlog against user licensing so we don't forget
https://tools.hmcts.net/jira/browse/SIDM-4391